elastic/apm-server
# [cloud] default to medium compression

## Motivation/summary

Set medium compression level (5) on cloud to retain current behavior when switching to apm server integration. 

## How to test these changes
* run `CLOUD_APM_CAPACITY=1024 ./apm-server export config` and check that no `output.elasticsearch.compression_level` is set
* run `CLOUD_APM_CAPACITY=1024 ./apm-server export config` and check that `output.elasticsearch.compression_level` is set to `5`
